### PR TITLE
Add meethod for activate-invoice endpoint

### LINKED ISSUE #50: Add support for manual activation of Walley invoices
**Is your feature request related to a problem? Please describe.**

Paytrail provides customer an option to pay with invoice. For certain invoice payment methods (currently only Walley/Collector), it is possible to activate the invoice manually later. This can be used for example with preordered products.

**Describe the solution you'd like**

Let's add support for this functionality: https://docs.paytrail.com/#/?id=manually-activating-invoices

### PR DESCRIPTION
**Pull request by VuorelaJ-P**
## Description

Add method for avtivate-invoice endpoint, https://docs.paytrail.com/#/?id=manually-activating-invoices

Fix #50

### CODE REVIEW

**Review by tommi-martin — APPROVED** [review]
Looks good, however do you need to add the support for manual invoice activation flag into this pull request as well?

**Comment by VuorelaJ-P** [comment]
> Looks good, however do you need to add the support for manual invoice activation flag into this pull request as well?

Good point, thought there already was said thing. Will be added to create payment request.

File: src/Request/AbstractPaymentRequest.php [hunk 1]
```diff
@@ -535,4 +542,15 @@ public function getGroups(): array
     {
         return $this->groups;
     }
+
+    public function setManualInvoiceActivation($value = false): PaymentRequestInterface
```
Comment (tommi-martin) [inline src/Request/AbstractPaymentRequest.php:546]: Method missing php Docs

File: src/Request/AbstractPaymentRequest.php [hunk 2]
```diff
@@ -535,4 +542,15 @@ public function getGroups(): array
...
+    {
+        $this->manualInvoiceActivation = $value;
+        return $this;
+    }
+
+    public function getManualInvoiceActivation(): bool
```
Comment (tommi-martin) [inline src/Request/AbstractPaymentRequest.php:552]: Method missing php Docs

**Review by tommi-martin — COMMENTED** [review]
A question relating to the setManualInvoiceActivation and getManualInvoiceActivation methods added: Should they be added to some other file instead of the abstract PaymentRequest? 
Method names refer to invoices yet when added to abstracts they will be available with all PaymentRequests.

**Pull request merged**